_progress-bar.scss

```diff
@@ -0,0 +1,252 @@
+// progress-bar
+// ===
+
+// Configurable Variables
+// ---
+
+$progress-bar__progress-color: black !default;
+$progress-bar__success-color: $success-color !default;
+$progress-bar__error-color: $error-color !default;
+$progress-bar__background-color: $neutral-50 !default;
+
+
+// Dependencies
+// ---
+
+// Override Font Awesome's default path
+$fa-font-path: '/bower_components/font-awesome/fonts';
```

Comment:
Can we put this and the `@import` into an if statement? That we we can configure the path to `null` or `false` and then not import the font awesome styles.